From: Andrew Morton <akpm@linux-foundation.org>
Subject: memcg-v1-no-need-for-memcg-locking-for-mglru-fix
Date: Fri Oct 25 06:28:06 PM PDT 2024

remove now-unused local:

mm/vmscan.c: In function 'walk_mm':
mm/vmscan.c:3652:28: error: unused variable 'memcg' [-Werror=unused-variable]
 3652 |         struct mem_cgroup *memcg = lruvec_memcg(lruvec);
      |                            ^~~~~
Cc: Hugh Dickins <hughd@google.com>
Cc: Johannes Weiner <hannes@cmpxchg.org>
Cc: Michal Hocko <mhocko@suse.com>
Cc: Muchun Song <muchun.song@linux.dev>
Cc: Roman Gushchin <roman.gushchin@linux.dev>
Cc: Shakeel Butt <shakeel.butt@linux.dev>
Cc: Yosry Ahmed <yosryahmed@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/vmscan.c |    2 --
 1 file changed, 2 deletions(-)

--- a/mm/vmscan.c~memcg-v1-no-need-for-memcg-locking-for-mglru-fix
+++ a/mm/vmscan.c
@@ -3646,10 +3646,8 @@ static void walk_mm(struct mm_struct *mm
 		.p4d_entry = walk_pud_range,
 		.walk_lock = PGWALK_RDLOCK,
 	};
-
 	int err;
 	struct lruvec *lruvec = walk->lruvec;
-	struct mem_cgroup *memcg = lruvec_memcg(lruvec);
 
 	walk->next_addr = FIRST_USER_ADDRESS;
 
_
